projects/core/src/lib/colors/colorizer.model.ts

```diff
@@ -96,13 +96,16 @@ export class LinearGradient extends Colorizer {
             return this.overColor;
         }
 
-        const index = this.breakpoints.findIndex((b) => value >= b.value);
+        const index = this.breakpoints.findIndex((b, i, brks) => {
+            return value >= b.value && (i === brks.length - 1 || value < brks[i + 1].value);
+        });
```

Comment:
Can't this be done by using `findLastIndex`?

https://jsfiddle.net/8yushjno/

Reply:
We shouldn't increase the javascript version to ES2023 if this is not recommended by Angular.

Maybe just reverse the array?

https://jsfiddle.net/m38y5416/